components/x-follow-button/src/FollowButton.jsx

```diff
@@ -0,0 +1,74 @@
+import { h } from '@financial-times/x-engine';
+import { withActions } from '@financial-times/x-interaction';
+
+import Input from './Input';
+import Button from './Button';
+
+import styles from './styles/main.scss';
+
+const followButtonActions = withActions((props) => ({
+	onSubmitAction(event) {
+		event.preventDefault();
+		console.log('the follow button was pressed');
+		console.log('the default event was prevented');
+		console.log(`the action suppose to trigger the submission of the form with "${event.target.method}" method and "${event.target.action}" action`);
```

Comment:
should these `console.log` be removed?

Reply:
The latest `console.log` allows to see that the parameters passed to the form are correct (when the default event was prevented) => when it would not, the form action would be correct (if I understand how it all works correctly)

Reply:
here's my feelings on this. i'm against shipping `console.log`s in a component, because they're not useful for an app apart from when testing.

the component wrapped with actions [is documented as the main entry point of this component](https://github.com/Financial-Times/x-dash/pull/59/files#diff-d57750bbbabfbdf1eddfecc685d3dda2R32), so it looks like you're intending people to use the actions.

if these actions aren't meant to be used by most people consuming the component, then i would advise removing them and allowing passing in `onSubmit` and `onClick` handlers for consumers that need them. if the actions are meant to be used, please remove the `console.log`s (or the entire `onSubmit` action).